docs/website/11ty/resolve-path.transform.js

```diff
@@ -29,9 +30,17 @@ class PathResolver {
   static resolveLinks(dom, filePath) {
     dom.querySelectorAll('a[href^="."]').forEach((link) => {
       const resolved = PathResolver.resolveLink(link.href, filePath);
+
       if (resolved !== link.href) console.info(blue(`Rewrite link "${link.href}" to "${resolved}"`));
       link.href = resolved;
     });
+
+    dom.querySelectorAll('a[href^="https:"]').forEach((link) => {
```

Comment:
As I can see, all internal links start with "/" (/content/.. ; /etoolbox-.../..)
Do we need another check?
Just by looking for /etoolbox-authoring-kit in url, github links will not open in new tab

Reply:
I think it's better to have it.
https is only a guarantee that the URL is absolute, but the current content doesn't mean that all the external links going to be relative.
We have the domain name in the env config, we can get the domain name from the link via URL utility.
 So we can check that the resource is "external" using the definition of "external" by comparing the domain name.
 Moreover, such a mechanism will be much more extendable in the future (wildcards, whitelist. blacklist if that will be required in the future).